app/models/foreman_tasks/concerns/action_triggering.rb

```diff
@@ -152,7 +152,7 @@ def execute_planned_action
           run = ::ForemanTasks.dynflow.world.execute(@execution_plan.id)
           if @dynflow_sync_action
             run.wait
-            if run.value.error?
+            if run.value.try(:error?)
```

Comment:
note for myself - this only happens when I do image based provisioning and I receive facts from chef-client